lib/infer.h

```diff
@@ -57,6 +56,4 @@ std::vector<ValueFlow::Value> infer(const ValuePtr<InferModel>& model,
 CPPCHECKLIB std::vector<MathLib::bigint> getMinValue(const ValuePtr<InferModel>& model, const std::list<ValueFlow::Value>& values);
 std::vector<MathLib::bigint> getMaxValue(const ValuePtr<InferModel>& model, const std::list<ValueFlow::Value>& values);
 
-std::string toString(const Interval& i);
```

Comment:
@pfultz2 it appears this function should have been called somewhere but it never way since it inception. Would have probably not been possible as it refers to an implementation-specific type.